pennylane/ops/qubit/observables.py

```diff
@@ -427,13 +420,28 @@ def __copy__(self):
 
         return copied_op
 
+    def __reduce__(self):
```

Comment:
this shouldn't be necessary for pickling anymore, right? the problem was type mismatches, but they should now be the same type and pickle should be happy

Reply:
Solved! (thanks ❤️)